packages/manager/modules/bm-server-components/src/netboot/translations/Messages_fr_FR.json

```diff
@@ -14,6 +14,7 @@
   "server_configuration_netboot_option_harddisk": "Booter sur le disque dur",
   "server_configuration_netboot_option_rescue": "Booter en mode rescue",
   "server_configuration_netboot_option_network": "Booter en mode network",
+  "server_configuration_netboot_option_internal": "Booter en mode réseau",
```

Comment:
Hmm are we sure of this content? I mean, if we have internal and network options displayed, we will have two sentences with the same meaning. That will be confusing

Reply:
I have asked the reporter. Let us wait for the confirmation.